src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindOperationWithSortIntegrationTest.java

```diff
@@ -93,18 +93,14 @@ public void sortWithSkipLimit() {
 
       JsonNodeFactory nodefactory = objectMapper.getNodeFactory();
       final ArrayNode arrayNode = nodefactory.arrayNode(10);
-      try {
-        for (int i = 0; i < 20; i++) {
-          if (i >= 10) {
-            arrayNode.add(
-                objectMapper.readTree(
-                    objectMapper
-                        .writerWithDefaultPrettyPrinter()
-                        .writeValueAsString(testDatas.get(i))));
-          }
+      for (int i = 0; i < 20; i++) {
```

Comment:
It would be great to have 21 documents here, or reduce `skip` to 9, just to make sure `limit` applies. Right now we're only testing that `skip` works, this test would pass even if `limit` was ignored.

Reply:
There are 25 documents in total inserted in the test..